internal/server/evaluator.go

```diff
@@ -23,6 +26,27 @@ func (s *Server) Evaluate(ctx context.Context, r *flipt.EvaluationRequest) (*fli
 	if err != nil {
 		return resp, err
 	}
+
+	spanAttrs := []attribute.KeyValue{
+		fliptotel.AttributeFlag.String(r.FlagKey),
+		fliptotel.AttributeEntityID.String(r.EntityId),
+		fliptotel.AttributeRequestID.String(r.RequestId),
+	}
+
+	if resp != nil {
+		spanAttrs = append(spanAttrs,
+			fliptotel.AttributeMatch.Bool(resp.Match),
+			fliptotel.AttributeSegment.String(resp.SegmentKey),
+			fliptotel.AttributeValue.String(resp.Value),
+			fliptotel.AttributeReason.String(resp.Reason.String()),
+			fliptotel.AttributeDurationMS.Float64(resp.RequestDurationMillis),
```

Comment:
Quick one: while I have you. Maybe this one should be dropped:
```suggestion
```

Duration is something which should be captured naturally through the span start and end timestamps. Also, it is something we dont really want to index like other attributes.